auditbeat/auditbeat.reference.yml

```diff
@@ -284,9 +297,9 @@ output.elasticsearch:
   # Optional load balance the events between the Logstash hosts
   #loadbalance: true
 
-  # Number of batches to be send asynchronously to logstash while processing
+  # Number of batches to be sent asynchronously to logstash while processing
   # new batches.
-  #pipelining: 0
+  #pipelining: 5
```

Comment:
Is there a reason we changed the default to 5? Should probably be mentioned in the changelog?